docs/source/reference/multi_objective/index.rst

```diff
@@ -3,6 +3,8 @@
 optuna.multi_objective
 ======================
 
+This module is deprecated, with former functionality moved to :class:`optuna.trial`, :class:`optuna.study`, :class:`optuna.samplers` and :class:`optuna.visualization`.
```

Comment:
A nit, but how about arranging the modules in the same order in the table?
In addition, we can use `:mod:` instead of `:class:` for the modules.

```suggestion
This module is deprecated, with former functionality moved to :mod:`optuna.samplers`, :mod:`optuna.study`, :mod:`optuna.trial` and :mod:`optuna.visualization`.
```